### PR TITLE
set a default root_directory so source code highlighting works out of…

### PR DESCRIPTION
**Pull request by grosser**
… the box

fixes https://github.com/airbrake/airbrake-ruby/issues/167

### CODE REVIEW

**Comment by kyrylo** [comment]
I'm happy to merge this when the Rubocop offence is fixed.

**Comment by grosser** [comment]
thx for pinging ... did not watch CI :D 
updated!

**Comment by kyrylo** [comment]
Thanks. Although Circle is stuck, I can see some failing tests :(

**Comment by grosser** [comment]
all good!

File: spec/airbrake_spec.rb [hunk 1]
```diff
@@ -167,11 +167,11 @@
       filter_chain = notifier.instance_variable_get(:@filter_chain)
       filters = filter_chain.instance_variable_get(:@filters)
 
-      expect(filters.size).to eq(2)
+      expect(filters.size).to eq(3)
```
Comment (grosser) [inline spec/airbrake_spec.rb]: idk why this happened ...

Comment (grosser) [inline spec/airbrake_spec.rb]: might be that a filter is added when root_directory exists

Comment (kyrylo) [inline spec/airbrake_spec.rb]: That's expected https://github.com/airbrake/airbrake-ruby/blob/43f780b2053532349056fb06e9814ea203beba4a/lib/airbrake-ruby/filter_chain.rb#L56-L57

**Pull request merged**